content/post/072-js-ipfs-0.39.md

```diff
@@ -0,0 +1,140 @@
+---
+date: 2019-10-24
+url: 071-js-ipfs-0-39
+title: js-ipfs 0.39.0 released
+author: Alex Potsides
+---
+
+# 🔦 Highlights
+
+> Configure your node with preset profiles! Limit outgoing browser requests! Start your node from fresh with one command!
+
+```console
+$ npm install -g ipfs
+```
+
+## 📋 Config profiles
```

Comment:
There _must_ be a more exciting emoji!

```suggestion
## 🛠 Config profiles
```

Reply:
You're the expert 🤣

Reply:
It's all Greek to me: https://www.nytimes.com/interactive/2014/07/25/style/emoji-quiz.html

Reply:
discuss.ipfs.io does not share your emoji enthusiasm:

![image](https://user-images.githubusercontent.com/665810/67396951-633c9d00-f5a0-11e9-8a72-eb839a3558e5.png)